app/views/ConfirmationView.scala.html

```diff
@@ -120,7 +120,7 @@ <h3 class="govuk-heading-s">@messages("confirmation.breakdown.dt")
         <li>@Html(messages("confirmation.nextSteps.li.4.1", link(appConfig.jobRetentionScheme, "confirmation.nextSteps.li.4.2", true)))</li>
     </ul>
 
-    @button(messages("confirmation.startAgain"), href = Some(routes.ClaimPeriodStartController.onPageLoad()), classes = Some("govuk-button"))
+    @button(messages("confirmation.startAgain"), href = Some(routes.ClaimPeriodQuestionController.onPageLoad()), classes = Some("govuk-button"))
```

Comment:
i think we need feature flag for this to make master releasable

Reply:
as above comment

Reply:
if you use feature flag in the confirmation.view to show either claim-start or claim-period-question then and  if the feature is disabled, then the  `claim-period-question` links/behaviour is completely hidden from the user and users wont touch this new urls at all

the pattern so far we followed is if the feature is disabled we show 404